src/views/credits/credits.jsx

```diff
@@ -2,458 +2,158 @@ const React = require('react');
 const render = require('../../lib/render.jsx');
 const FormattedHTMLMessage = require('react-intl').FormattedHTMLMessage;
```

Comment:
`FormattedHTMLMessage` has been deprecated. Whenever we update strings that use it, we replace them with normal `FormattedMessage` strings with placeholders. The wiki has [instructions for doing the conversion.](https://github.com/LLK/scratch-www/wiki/Localization-Guide#removing-formattedhtmlmessage)